ui/src/user/components/__tests__/SignUpPage.test.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import SignUpPage from '../SignUpPage';
+import { Formik } from 'formik';
+
+describe('SignUpPage', () => {
+  it('renders page', () => {
+    const wrapper = shallow(
+      <SignUpPage loading={false} onSubmit={jest.fn()} />
+    );
+    const formWrapper = wrapper.find(Formik);
+    expect(wrapper).toMatchSnapshot();
+    expect(formWrapper.dive()).toMatchSnapshot();
+  });
+
+  it('renders page with loading', () => {
+    const wrapper = shallow(<SignUpPage loading onSubmit={jest.fn()} />);
+    const formWrapper = wrapper.find(Formik);
+
+    expect(wrapper).toMatchSnapshot();
+    expect(formWrapper.dive()).toMatchSnapshot();
```

Comment:
it's OK like that, but usually it's better to separate this to another component (`SignUpForm`) when test gets like this.